docs/en/developement/NewLicenseGuide.md

```diff
@@ -2,9 +2,19 @@
 
 If you have any new Jar binary package adding in you PR, you need to follow the steps below to notice license
 
-1. declared in `tools/dependencies/konw-dependencies.txt`,
-2. Add the corresponding License file under `seatunnel-dist/release-docs/licenses`, if it is a standard Apache License, it does not need to be added.
+1. declared in `tools/dependencies/konw-dependencies.txt`
+
+2. Add the corresponding License file under `seatunnel-dist/release-docs/licenses`, if it is a standard Apache License, it does not need to be added
+
 3. Add the corresponding statement in `seatunnel-dist/release-docs/LICENSE`
+
+   ```bash
+   # At the same time, you can also use the script to assist the inspection.
+   # Because it only uses the Python native APIs and does not depend on any third-party libraries, it can run using the original Python environment.
+   # Please refer to the documentation if you do not have a Python env: https://www.python.org/downloads/
+   python3 bin/license.py seatunnel-dist/target/THIRD-PARTY.txt seatunnel-dist/release-docs/LICENSE true
```

Comment:
I don't know much about the license, but what is it file `seatunnel-dist/target/THIRD-PARTY.txt` look like and how it generated

Reply:
I have described in the related issues, you can get the details from there.

Reply:
Also thanks for your mention, I just add more details to the relate doc as well.